backend/src/utils/auth.py

```
@@ -2,12 +2,14 @@
 Authentication utility functions.
 """
 import logging
-from typing import Dict, Any, Optional
+from typing import Dict, Any, Optional, Callable
+from functools import wraps
 
 from models.file_map import FieldMapping, FileMap
```

Ruff (0.12.2)

`models.file_map.FieldMapping` imported but unused

Remove unused import

(F401)